doc/nrf/glossary.rst

```diff
@@ -10,6 +10,9 @@ Glossary
 .. glossary::
    :sorted:
 
+   Accessory device
+      In :ref:`ug_matter`, a network node that contains the necessary firmware to be commissioned into a thread mesh network.
```

Comment:
Although we use "accessory" in some our docs, in Matter itself it's more common to use just "Matter device". Maybe we should switch to the latter, too? Also, a device is not equivalent to a node as a single device can contain multiple nodes. Should we change it to: "A physical equipment with an appropriate firmware that allows to create a single or several Matter nodes"?

Matter node can then be defiened as: "An element of a Matter network having a unique ID, network address and operational credentials"